README.md

```diff
@@ -1,186 +1,246 @@
-[![poliastro](https://www.poliastro.space/images/logo_text.png)](https://www.poliastro.space/)
-
-Name:         | poliastro
-:------------:|:--------------
-**Website**:  | <https://www.poliastro.space/>
-**Author**:   | Juan Luis Cano Rodríguez [![orcid](https://img.shields.io/badge/id-0000--0002--2187--161X-a6ce39.svg)](http://orcid.org/0000-0002-2187-161X)
-**Version**:  | 0.18.dev0
-
-[![poliastro](https://img.shields.io/circleci/build/gh/poliastro/poliastro/main?style=flat-square)](https://circleci.com/gh/poliastro/poliastro/?branch=main)
-[![codecov](https://img.shields.io/codecov/c/github/poliastro/poliastro.svg?style=flat-square)](https://codecov.io/github/poliastro/poliastro?branch=main)
-[![pre-commit.ci status](https://results.pre-commit.ci/badge/github/poliastro/poliastro/main.svg)](https://results.pre-commit.ci/latest/github/poliastro/poliastro/main)
-
-[![docs](https://img.shields.io/badge/docs-latest-brightgreen.svg?style=flat-square)](https://docs.poliastro.space/en/latest/?badge=latest)
-[![license](https://img.shields.io/badge/license-MIT-blue.svg?style=flat-square)](https://github.com/poliastro/poliastro/raw/main/COPYING)
-[![doi](https://zenodo.org/badge/11178845.svg?style=flat-square)](https://zenodo.org/badge/latestdoi/11178845)
-[![astropy](http://img.shields.io/badge/powered%20by-AstroPy-orange.svg?style=flat-square)](http://www.astropy.org/)
-[![mailing](https://img.shields.io/badge/mailing%20list-groups.io-8cbcd1.svg?style=flat-square)](https://groups.io/g/poliastro-dev)
-[![Join the chat at http://chat.poliastro.space/](https://img.shields.io/matrix/poliastro:matrix.org.svg?style=flat-square)](http://chat.poliastro.space/)
-
-[![OpenCollective](https://opencollective.com/poliastro/backers/badge.svg)](#backers)
-[![OpenCollective](https://opencollective.com/poliastro/sponsors/badge.svg)](#sponsors)
-
-poliastro is an open source (MIT) pure Python library
-for interactive Astrodynamics and Orbital Mechanics,
-with a focus on ease of use, speed, and quick visualization.
-It provides a simple and intuitive API,
-and handles physical quantities with units.
-
-Some features include
-orbit propagation, solution of the Lambert\'s problem,
+[![poliastro Logo](https://raw.githubusercontent.com/jorgepiloto/poliastro/fix/molniya-plot/docs/source/_static/logo_readme.png)](https://docs.poliastro.space/en/stable/)
```

Comment:
```suggestion
[![poliastro Logo](https://raw.githubusercontent.com/poliastro/poliastro/main/docs/source/_static/logo_readme.png)](https://docs.poliastro.space/en/stable/)
```

It will be broken in the preview, but needs to change before landing in `main`